src/index.ts

```diff
@@ -7,14 +7,12 @@ export {
     Types,
     TypedValues,
     TypedData,
-    TypedDataOptions,
+    ITypedDataOptions,
```

Comment:
What is the profit in that renaming that is worthy of potential breaking client libraries?

Reply:
By the way, there is a special 'refactor' type of commit.

Reply:
I suppose that impact of these changes is minimal and the refactoring is about fixing of some interface names